package.json

```
@@ -66,42 +73,44 @@
     "ember-cli-terser": "^4.0.2",
     "ember-composable-helpers": "^5.0.0",
     "ember-concurrency": "^2.2.1",
-    "ember-data": "~3.28.6",
-    "ember-export-application-global": "^2.0.1",
-    "ember-fetch": "^8.1.1",
+    "ember-data": "~4.11.3",
```

```suggestion
    "ember-data": "~4.12.3",
```

I think Ember Data 4.12 was released slightly to late, so they missed the blueprint bump..